packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -272,8 +272,6 @@ const VirtualListBaseFactory = (type) => {
 		}
 
 		componentDidUpdate (prevProps) {
-			this.hasDataSizeChanged = (prevProps.dataSize !== this.props.dataSize);
```

Comment:
`this.hasDataSizeChanged` was used in `componentDidUpdate` function of `ui/Scrollable.js` and `ui/ScrollableNative.js` to know the timing when a VirtualList have finished rendering at last.

Reply:
Needless to say, this is a terrible way of handling state change. I will revert the change, but we need to clean up this mess.